cmd/bosun/conf/conf.go

```diff
@@ -111,6 +112,14 @@ func (c *Conf) GraphiteContext() graphite.Context {
 	return graphite.Host(c.GraphiteHost)
 }
 
+// LogstashContext returns a Logstash context
+func (c *Conf) LogstashContext() logstash.Context {
+	if len(c.LogstashElasticHosts) < 1 {
+		return nil
```

Comment:
Are we checking for nil everywhere we create one of these? Perhaps it would be better to return a no-op instance rather than nil?